rebar.config

```diff
@@ -1,31 +1,29 @@
 %-*-Erlang-*-
-{erl_opts, [debug_info, {parse_transform, lager_transform}]}.
+{erl_opts, [debug_info]}.
 
 {deps, [
-    {setup,         "2.0.2"},
-    {lager,         "3.6.8"},
-    {exometer_core, "1.5.2"},
+    {setup,         {git, "https://github.com/uwiger/setup.git", {branch, "master"}}},
+    {exometer_core, "1.5.7"},
     {exometer_influxdb, "0.6.0"},
-    {erlando,       {git, "https://github.com/travelping/erlando.git", {tag, "1.0.2"}}},
+    {erlando,       {git, "https://github.com/travelping/erlando.git", {branch, "master"}}},
     {dtlsex,        {git, "git://github.com/RoadRunnr/dtlsex.git", {branch, "master"}}},
     {regine,        {git, "git://github.com/travelping/regine.git", {branch, "master"}}},
-    {ergw_aaa,      {git, "git://github.com/travelping/ergw_aaa.git", {tag, "2.2.0"}}},
-    {gen_socket,    {git, "git://github.com/travelping/gen_socket.git", {branch, "master"}}},
+    {ergw_aaa,      {git, "git://github.com/travelping/ergw_aaa", {tag, "master"}}},
     {pcapng,        {git, "git://github.com/travelping/pcapng.git", {branch, "master"}}},
-    {cowboy,        "2.6.1"},
+    {cowboy,        "2.8.0"},
     {pbkdf2,        "2.0.0"},
-    {jsx,           "2.9.0"},
+    {jsx,           "3.0.0"},
     {parse_trans,   "3.3.0"},
```

Comment:
Is this needed? I saw it removed from line 2.

Reply:
yes, jsx is still used